src/ng/compile.js

```diff
@@ -3131,7 +3131,13 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
               childBoundTranscludeFn);
           }
           linkQueue = null;
-        }).catch(noop);
+        }).
+        catch(function(error) {
+          if (error instanceof Error) {
+            $exceptionHandler(error);
+          }
+        }).
+        catch(noop);
```

Comment:
This catch shouldn't be needed.  The one above it is already catching all errors and is not re-throwing or returning a rejected promise, so this should be unreachable.